### PR TITLE
feat: add the option to add dapp icon in base64 format

### PR DESCRIPTION
**Pull request by omridan159**
Add the option to add Dapp icon in base64 format

### CODE REVIEW

File: packages/sdk-communication-layer/src/services/RemoteCommunication/ConnectionManager/initCommunicationLayer.ts [hunk 1]
```diff
@@ -94,7 +94,7 @@ export function initCommunicationLayer({
     url,
     title,
     source: state.dappMetadata?.source,
-    icon: state.dappMetadata?.iconUrl,
+    icon: state.dappMetadata?.iconUrl || state.dappMetadata?.base64Icon,
```
Comment (andreahaku) [inline packages/sdk-communication-layer/src/services/RemoteCommunication/ConnectionManager/initCommunicationLayer.ts:97]: please name this property `icon` to keep compatibility with what's it's name already

**Review by andreahaku — COMMENTED** [review]
keep the existing name please :) thanks

File: packages/sdk-communication-layer/src/services/RemoteCommunication/ConnectionManager/initCommunicationLayer.ts (hunk 1 above)
Comment (omridan159) [reply packages/sdk-communication-layer/src/services/RemoteCommunication/ConnectionManager/initCommunicationLayer.ts:97]: `base64Icon` was it's name before it got removed

Comment (andreahaku) [reply packages/sdk-communication-layer/src/services/RemoteCommunication/ConnectionManager/initCommunicationLayer.ts:97]: my fault, it was base64Icon :(

**Review by andreahaku — APPROVED** [review]
LGTM

**Review by andreahaku — APPROVED** [review]
LGTM

**Pull request merged**